external-api/src/http/admin.rs

```diff
@@ -37,7 +38,21 @@ pub struct IsLeaderResponse {
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct OpenOrdersResponse {
     /// The open orders
-    pub orders: Vec<OrderMetadata>,
+    pub orders: Vec<OpenOrder>,
+}
+
+/// An open order, containing the order's metadata
+/// as well as the fillable amount of the order
+/// given the underlying wallet's balances and potentially
```

Comment:
nit: why potentially? `price` is not optional

Reply:
ah poor wording on my behalf, the implication here is that only for buy orders does the price factor into the fillable amount. clearer to just remove "potentially".